src/components/mgt-people/mgt-people.ts

```diff
@@ -146,6 +158,9 @@ export class MgtPeople extends MgtTemplatedComponent {
   private _userIds: string[];
   private _peopleQueries: string[];
 
+  // tslint:disable-next-line:completed-docs
+  private _peoplePresence: Array<{ '@odata.context': string; id: string; availability: string; activity: string }>;
```

Comment:
should this type just be `Presence`?

Reply:
It's Presence plus the '@data.context' part because of the search.